packages/matchbox/src/components/Slider/Slider.js

```diff
@@ -151,64 +198,63 @@ function Slider(props) {
         window.removeEventListener('touchmove', handleTouchMove);
         window.removeEventListener('touchend', handleEnd);
       }
-    });
+    };
   }, [moving]);
 
-  const sliderClasses = classnames(
-    styles.Slider,
-    disabled && styles.Disabled
-  );
-
-  const tickMarkup = Object.keys(tickLocations).map((tick) => {
+  const tickMarkup = Object.keys(tickLocations).map(tick => {
     const { label, position } = tickLocations[tick];
     return (
-      <div
+      <StyledTick
         key={tick}
         style={{ left: position }}
-        className={classnames(styles.Tick, tick < sliderValue && styles.Included)}>
-        <div className={styles.TickLabel}>{label}</div>
-      </div>
+        disabled={disabled}
+        included={tick < sliderValue}
+      >
+        <StyledTickLabel>{label}</StyledTickLabel>
+      </StyledTick>
     );
   });
 
   return (
-    <div
-      className={sliderClasses}
-      data-id='slider-wrapper'
-      onTouchStart={disabled ? noop : handleTouchStart}
-      onMouseDown={disabled ? noop : handleMouseDown}
-      ref={sliderRef}
-    >
-      <div className={styles.Rail} />
-      {tickMarkup}
-      <div
-        className={styles.Track}
-        style={{ width: sliderLocation }}
-      />
-      <div
-        id={id}
-        aria-controls={props['aria-controls']}
-        aria-valuemin={min}
-        aria-valuemax={max}
-        aria-valuenow={sliderValue}
-        aria-disabled={disabled}
-        className={styles.Handle}
-        data-id={props['data-id']}
-        onBlur={onBlur}
-        onFocus={onFocus}
-        onKeyDown={disabled ? noop : handleKeyDown}
-        role='slider'
-        style={{ left: sliderLocation }}
-        tabIndex='0'
-      />
-    </div>
+    <Box pb={400}>
```

Comment:
Did you mean to add this wrapper? I think cypress tests _might_ be failing cause of this additional wrapper. Total height here should also be 40px, Ill comment where you can set that instead of here

<img width="335" alt="Screen Shot 2020-03-09 at 9 02 35 AM" src="https://user-images.githubusercontent.com/3903325/76215301-d9a30a00-61e4-11ea-8cd1-52f602f89618.png">

Reply:
The reason I added a wrapper was to allow padding for the tick labels, but if we don't need this I'll remove it and we can just use system props for margin on the slider.
<img width="1016" alt="Screen Shot 2020-03-09 at 9 39 24 AM" src="https://user-images.githubusercontent.com/3878251/76227257-06f3b600-61ed-11ea-9c9a-1650251db075.png">